### PR TITLE
Scale in a thread

### PR DESCRIPTION
**Pull request by obino**

### CODE REVIEW

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/2652/

**Comment by menivaitsi** [comment]
@obino Please add the build after last commit.

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/2715/

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/2716/

File: AppController/lib/haproxy.rb [hunk 1]
```diff
@@ -136,6 +136,12 @@ def self.create_tq_endpoint_config(server_ips, my_ip, listen_port)
     self.create_app_config(servers, my_ip, listen_port, TaskQueue::REST_NAME)
   end
 
+  # Remove the configuration for Tasqueue REST API endpoints.
```
Comment (tmarballi) [inline AppController/lib/haproxy.rb]: Tasqueue -> Taskqueue

**Pull request merged**